lib/zendesk_apps_support/manifest.rb

```diff
@@ -50,30 +42,31 @@ def no_template_locations
     end
 
     def location?
-      locations != { 'zendesk' => {} }
+      !locations.values.all?(&:empty?)
     end
 
     def locations
       @locations ||=
         case original_locations
         when Hash
-          original_locations
+          replace_legacy_locations! original_locations
```

Comment:
This mutates original locations, right? I guess we're not currently using `original_locations` but would it be hard to make a copy instead? Otherwise if we eventually use `original_locations` someone's going to get a surprise ;)